packages/libjson-c/libjson-c.spec

```diff
@@ -0,0 +1,53 @@
+%global jsonc_ver 0.18
+%global jsonc_rel 20240915
+
+Name: %{_cross_os}libjson-c
+Version: %{jsonc_ver}
+Release: %{jsonc_rel}%{?dist}
+Summary: Library for JSON
+License: MIT
+Source0: https://github.com/json-c/json-c/archive/refs/tags/json-c-%{jsonc_ver}-%{jsonc_rel}.tar.gz
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using the library for JSON
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n json-c-json-c-%{jsonc_ver}-%{jsonc_rel} -p1
+
+%build
+%{cross_cmake} . \
+  -DBUILD_APPS:BOOL=OFF \
+  -DBUILD_SHARED_LIBS:BOOL=ON \
+  -DBUILD_STATIC_LIBS:BOOL=OFF \
+  -DDISABLE_BSYMBOLIC:BOOL=OFF \
+  -DDISABLE_EXTRA_LIBS:BOOL=ON \
+  -DDISABLE_JSON_POINTER:BOOL=ON \
+  -DDISABLE_THREAD_LOCAL_STORAGE:BOOL=OFF \
+  -DDISABLE_WERROR:BOOL=OFF \
+  -DENABLE_RDRAND:BOOL=ON \
+  -DENABLE_THREADING:BOOL=ON \
+  -DCMAKE_INSTALL_PREFIX:PATH=%{_cross_prefix} \
```

Comment:
Also not needed:

https://github.com/bottlerocket-os/bottlerocket-sdk/blob/8ef6f479f9ea4d2f0f8e0552580cda07282ce706/macros/shared#L163

Reply:
> Also not needed:

If I don't set this, then I don't get the right behavior; `cmake --install` sends the files to `/usr/local` instead:
```
  #14 5.950 + DESTDIR=/home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64
  #14 5.950 + cmake --install .
  #14 5.960 -- Install configuration: "Release"
  #14 5.961 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/lib/libjson-c.so.5.4.0
  #14 5.961 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/lib/libjson-c.so.5
  #14 5.961 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/lib/libjson-c.so
  #14 5.962 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/lib/cmake/json-c/json-c-targets.cmake
  #14 5.962 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/lib/cmake/json-c/json-c-targets-release.cmake
  #14 5.962 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/lib/cmake/json-c/json-c-config.cmake
  #14 5.963 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/lib/pkgconfig/json-c.pc
  #14 5.963 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_config.h
  #14 5.963 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json.h
  #14 5.963 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/arraylist.h
  #14 5.964 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/debug.h
  #14 5.964 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_c_version.h
  #14 5.964 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_inttypes.h
  #14 5.964 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_object.h
  #14 5.964 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_object_iterator.h
  #14 5.965 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_tokener.h
  #14 5.965 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_types.h
  #14 5.965 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_util.h
  #14 5.965 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/json_visit.h
  #14 5.965 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/linkhash.h
  #14 5.966 -- Installing: /home/builder/rpmbuild/BUILDROOT/bottlerocket-libjson-c-0.18-20240915.1731694836.c940f6ab.br1.aarch64/usr/local/include/json-c/printbuf.h
```

Reply:
:melting_face: , why cmake? I wonder if there is a bug in `cmake` and it ignores `CMAKE_INSTALL_PREFIX` when set through the `toolchain` file?